src/Model.php

```diff
@@ -43,6 +43,16 @@ abstract public function getKeyName();
      */
     abstract public function getColumns();
 
+    /**
+     * Get the configured table alias. (Default tablename)
+     *
+     * @return string
+     */
+    public function getTableAlias()
```

Comment:
Please add a return type hint.

Reply:
Do you mean return type declaration? Cause I don' t get what you mean by that. If you are referring to the return type declaration, we don't have a return type declaration anywhere else in all `ipl_*` either.